go.mod

```diff
@@ -147,16 +149,16 @@ require (
 	github.com/CycloneDX/cyclonedx-go v0.8.0
 	github.com/DataDog/appsec-internal-go v1.7.0
 	github.com/DataDog/datadog-agent/pkg/gohai v0.56.0-rc.3
-	github.com/DataDog/datadog-agent/pkg/obfuscate v0.56.0-rc.3
-	github.com/DataDog/datadog-agent/pkg/remoteconfig/state v0.56.0-rc.3
+	github.com/DataDog/datadog-agent/pkg/obfuscate v0.59.0-rc.1
+	github.com/DataDog/datadog-agent/pkg/remoteconfig/state v0.59.0-rc.1
 	github.com/DataDog/datadog-agent/pkg/security/secl v0.56.0-rc.3
 	github.com/DataDog/datadog-agent/pkg/trace v0.56.0-rc.3
 	github.com/DataDog/datadog-agent/pkg/util/cgroups v0.56.0-rc.3
-	github.com/DataDog/datadog-agent/pkg/util/log v0.57.1
+	github.com/DataDog/datadog-agent/pkg/util/log v0.59.0-rc.1
 	github.com/DataDog/datadog-agent/pkg/util/pointer v0.57.1
-	github.com/DataDog/datadog-agent/pkg/util/scrubber v0.57.1
+	github.com/DataDog/datadog-agent/pkg/util/scrubber v0.59.0-rc.1
 	github.com/DataDog/datadog-go/v5 v5.5.0
-	github.com/DataDog/datadog-operator v1.8.0-rc.1
+	github.com/DataDog/datadog-operator v0.7.1-0.20241010110733-dbbe6d120655
```

Comment:
This is the commit of DataDog/datadog-operator#1445.

To be updated as soon as this PR is merged.